src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -220,6 +220,7 @@ impl DragonballInner {
             max_vcpu_count: self.config.cpu_info.default_maxvcpus as u8,
             mem_type,
             mem_file_path,
+            pci_hotplug_enabled: true,
```

Comment:
Do we need to read from config?

Reply:
There's no need to get the value from the config, as it's just dragonball's behaviour. And we can do it just initialize it with `true`.